packages/docusaurus/src/server/config.ts

```diff
@@ -23,6 +23,7 @@ const OPTIONAL_FIELDS = [
   'themes',
   'presets',
   'themeConfig',
+  'image',
```

Comment:
add it as themeconfig as this is theme-classic specific instead of docusaurus.config